tests/skip_presets.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from tests.utils import (
+    is_os_windows_and_py_lt_3_8,
+    is_os_windows_and_py_lt_3_9,
+)
+
+skip_for_windows_py_lt_3_9 = pytest.mark.skipif(
```

Comment:
The same q as above, does it make sense to keep these in `conftest.py`? Except for introducing one more file. @mike0sv ?